Dockerfile.hpu.ubi

```diff
@@ -0,0 +1,119 @@
+ARG BASE_IMAGE=vault.habana.ai/gaudi-docker/1.18.0/rhel9.4/habanalabs/pytorch-installer-2.4.0:1.18.0-524
+FROM ${BASE_IMAGE} as habana-base
+
+USER root
+
+ENV VLLM_TARGET_DEVICE="hpu"
+ENV HABANA_SOFTWARE_VERSION="1.18.0-524"
+
+RUN dnf -y update --best --allowerasing --skip-broken && dnf clean all
+
+WORKDIR /workspace
+
+## Python Installer #################################################################
+FROM habana-base as python-install
+
+ARG PYTHON_VERSION=3.11
+
```

Comment:
Yes, that ENV var is used later on

Reply:
My point is that we do not need to reinstall Python-3.11 here. since it has already been installed in the base image see https://github.com/HabanaAI/Setup_and_Install/blob/5ec0d0c9be8faf9120bfc72d27bdff1eb5abec4f/dockerfiles/base/Dockerfile.rhel9.4#L67

Reply:
Ah, true